configure.ac

```diff
@@ -7206,6 +7203,15 @@ do
     ENABLED_SP_ECC=yes
     ENABLED_SP_SAKKE_1024=yes
     ;;
+  smallsm2)
```

Comment:
ran into an issue when trying to build with smallsm2 (maybe is in master too with just sp small?)

```
./configure --enable-all --enable-sp=smallsm2 --enable-sp-asm CC="clang -fsanitize=address" --enable-sm2 --enable-sm3 && make 
```

```
  CC       wolfcrypt/src/src_libwolfssl_la-ed25519.lo
wolfcrypt/src/eccsi.c:1437:45: error: unused parameter 'key' [-Werror,-Wunused-parameter]
static int eccsi_mulmod_point_add(EccsiKey* key, const mp_int* n,
                                            ^
wolfcrypt/src/eccsi.c:1437:64: error: unused parameter 'n' [-Werror,-Wunused-parameter]
static int eccsi_mulmod_point_add(EccsiKey* key, const mp_int* n,
                                                               ^
wolfcrypt/src/eccsi.c:1438:20: error: unused parameter 'point' [-Werror,-Wunused-parameter]
        ecc_point* point, ecc_point* a, ecc_point* res, mp_digit mp, int map)
                   ^
wolfcrypt/src/eccsi.c:1438:38: error: unused parameter 'a' [-Werror,-Wunused-parameter]
        ecc_point* point, ecc_point* a, ecc_point* res, mp_digit mp, int map)
                                     ^
wolfcrypt/src/eccsi.c:1438:52: error: unused parameter 'res' [-Werror,-Wunused-parameter]
        ecc_point* point, ecc_point* a, ecc_point* res, mp_digit mp, int map)
                                                   ^
wolfcrypt/src/eccsi.c:1438:66: error: unused parameter 'mp' [-Werror,-Wunused-parameter]
        ecc_point* point, ecc_point* a, ecc_point* res, mp_digit mp, int map)
                                                                 ^
wolfcrypt/src/eccsi.c:1438:74: error: unused parameter 'map' [-Werror,-Wunused-parameter]
        ecc_point* point, ecc_point* a, ecc_point* res, mp_digit mp, int map)
                                                                         ^
wolfcrypt/src/eccsi.c:1468:1: error: control reaches end of non-void function [-Werror,-Wreturn-type]
}
^
8 errors generated.
```

Reply:
Fixed